index.js

```diff
@@ -329,10 +329,10 @@ function showSidebar(sidebarProps) {
                         showSearch();
                         break;
                     case 'smarton':
-                        tabs.open('https://www.mozilla.org/teach/smarton/security/');
+                        tabs.open('https://www.mozilla.org/teach/smarton/security/?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1&amp;utm_content=security/');
                         break;
                     case 'newsletter':
-                        tabs.open('https://www.mozilla.org/#newsletter-subscribe');
+                        tabs.open('https://www.mozilla.org/?utm_source=fxonboarding&amp;utm_medium=firefox-browser&amp;utm_campaign=onboardingv1&amp;utm_content=community#newsletter-subscribe');
```

Comment:
@FrancescoSTL the newsletter URL is wrong given #208 do just want to handle the smartone link with this issue first?